src/app/resolvers/tech-record-validate/tech-record-validate.resolver.ts

```diff
@@ -8,20 +8,20 @@ import { HgvPsvVehicleConfiguration, TrlVehicleConfiguration } from '@models/veh
 import { Store } from '@ngrx/store';
 import { State } from '@store/.';
 import { selectTechRecord, updateEditingTechRecord } from '@store/technical-records';
+import * as _ from 'lodash';
```

Comment:
I don't think we should import this way from lodash if possible not to. Import via function